File: src/main/java/me/skynda/vehicle/entities/VehicleImage.java
package me.skynda.vehicle.entities;

import lombok.Data;
import me.skynda.common.entity.Image;

import javax.persistence.*;
import java.io.Serializable;
import java.util.Date;

@Entity
@Data
@Table(name = "vehicle_image")
public class VehicleImage implements Serializable{

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name="archived")
    private Date archived;

    @Column(name = "vehicle_id", nullable = false)
    private Integer vehicleId;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "image_id", nullable = false)
    private Image image;
}

File: skynda_dotnet_backend/Triven.Data.EntityFramework/Entities/VehicleImage.cs
using System.ComponentModel.DataAnnotations.Schema;
using Triven.Data.EntityFramework.Entities.Base;
using Triven.Domain.Models;

namespace Triven.Data.EntityFramework.Entities
{
    [Table("VehicleImage")]
    public class VehicleImage : AuditableModel, IVehicleImage
    {
        public int VehicleId { get; set; }
        public virtual Vehicle Vehicle { get; set; }

        public int ImageId { get; set; }
        public Image Image { get; set; }
    }
}